guides/common/modules/con_upgrading-el-on-project-or-proxy.adoc

```diff
@@ -4,9 +4,9 @@
 {Project} and {SmartProxy} are supported on both {EL} 8 and {EL} 9.
 You can use the following methods to upgrade your {Project} and {SmartProxy} operating system from {EL} 8 to {EL} 9:
 
-* Leapp in-place upgrade
-* Migration by using backup and restore
 * Migration by using cloning
+* Migration by using backup and restore
+* Leapp in-place upgrade
```

Comment:
Why is in-place last now? While I can see "clone before restore", I don't see "clone before leapp" -- especially as clone doesn't work for proxies but the entry here is "you can use the following methods for … proxy"

Reply:
Hey @evgeni I have reordered the introduction and sequence of chapters. Does seem better now?